test-suite/js/scalajs-test-suite-2.10-fastopt.html

```diff
@@ -1,17 +1,16 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Scala.js Test Suite - Jasmine HTML reporter</title>
+  <title>Scala.js Test Suite</title>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-  <link rel="stylesheet" type="text/css" href="http://cdn.jsdelivr.net/jasmine/1.3.1/jasmine.css"></link>
+  <link rel="stylesheet" type="text/css" href="test-stylesheet.css">
+  <script type="text/javascript">
+    var __ScalaJSEnv = {"javaSystemProperties": { "scalajs.fastopt-stage": "true" }};
+  </script>
+  <script type="text/javascript" src="./target/scala-2.10/scalajs-test-suite-test-jsdeps.js"></script>
+  <script type="text/javascript" src="./target/scala-2.10/test_definitions.js"></script>
```

Comment:
At https://github.com/scala-js/scala-js/pull/2316/files#diff-215113124f1de02f228327ba7abb45f7R565 you add `test_definitions.js` to the `jsDependencies`, but here you include `jsdeps` + `test_definitions`. That does not look good.